<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iCharger</class>
 <widget class="QWidget" name="iCharger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iCharger</string>
  </property>
  <widget class="QwtPlot" name="current">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>400</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="voltage_all">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>400</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="charged">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>400</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtThermo" name="intTemp">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>46</y>
     <width>52</width>
     <height>181</height>
    </rect>
   </property>
   <property name="alarmLevel">
    <double>40.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>70.000000000000000</double>
   </property>
  </widget>
  <widget class="QwtThermo" name="battTemp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>46</y>
     <width>52</width>
     <height>181</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="alarmLevel">
    <double>40.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>70.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge Current:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Charged Energy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage Levels Cells</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="cellVoltages">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Cell Voltages</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>192</width>
     <height>203</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current State:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="state">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current Channel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="channel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Timestamp:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="timestamp">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Input Voltage:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="inputV">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Battery Voltage:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="batteryV">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Current Current:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="current_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Charged:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="charged_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <double>999999.989999999990687</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>11</y>
     <width>79</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Internal Temperature</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>11</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Battery Temperature:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QwtPlot" name="voltage_cell">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>400</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage Level</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
